concepticondata/conceptlists/Gerardi-2020-244.tsv

```diff
@@ -0,0 +1,245 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	PORTUGUESE
+Gerardi-2020-244-1	1	ALL	98	ALL	TODOS / TUDO
+Gerardi-2020-244-2	2	ARM	1673	ARM	BRAÇO
+Gerardi-2020-244-3	3	ARROW	977	ARROW	FLECHA
+Gerardi-2020-244-4	4	ASH	646	ASH	CINZA(S)
+Gerardi-2020-244-5	5	AXE	677	AXE	MACHADO
+Gerardi-2020-244-6	6	BACK	1291	BACK	COSTAS
+Gerardi-2020-244-7	7	BAD/EVIL	3764	BAD OR EVIL	MAU / MAL / RUIM
+Gerardi-2020-244-8	8	BAMBOO	1927	BAMBOO	TAQUARA / BAMBU
+Gerardi-2020-244-9	9	BANANA	868	BANANA	BANANA
+Gerardi-2020-244-10	10	BASKET	1539	BASKET	CESTO / CESTA
+Gerardi-2020-244-11	11	BAT	1793	BAT	MORCEGO
+Gerardi-2020-244-12	12	BE_WRONG	1390	WRONG	ERRAR / ESTAR ERRADO
+Gerardi-2020-244-13	13	BEANS	2574	BEAN OR PEA	FEIJÃO
+Gerardi-2020-244-14	14	BEARD	678	BEARD	BARBA
+Gerardi-2020-244-15	15	BELLY	1838	NAVEL	UMBIGO
+Gerardi-2020-244-16	16	BREAK	680	BREAK (DESTROY OR GET DESTROYED)	QUEBRAR / ROMPER
+Gerardi-2020-244-17	17	BIG	1202	BIG	GRANDE
+Gerardi-2020-244-18	18	BIRD	937	BIRD	PÁSSARO
+Gerardi-2020-244-19	19	BITE	1403	BITE	MORDER
+Gerardi-2020-244-20	20	BLACK	163	BLACK	PRETO
+Gerardi-2020-244-21	21	BLOW	176	BLOW (WITH MOUTH)	SOPRAR
```

Comment:
You can use "wind" as the agent of the verb, but it is not that common; usually you'd use the noun ("o sopro do vento" -- "the wind's blow"). But I guess it would not be wrong, I think they colexify in most languages. @LanguageStructure ?

Reply:
@LanguageStructure there are two main concepts for "BLOW": BLOW (WITH MOUTH) and BLOW (OF WIND). The strategy in Concepticon has been to map to BLOW (OF WIND) when the same word is used.

I mapped to BLOW (WITH MOUTH) because most of the data is cognate with Old Tupi "peîu", which is glossed as "assoprar, soprar, abanar", and the examples I could find are related to the mouth. But does it apply to wind as well?